doc/man/bitcoin-qt.1

```diff
@@ -179,10 +179,6 @@ Allow DNS lookups for \fB\-addnode\fR, \fB\-seednode\fR and \fB\-connect\fR (def
 Query for peer addresses via DNS lookup, if low on addresses (default: 1
 unless \fB\-connect\fR used)
 .HP
-\fB\-enablebip61\fR
```

Comment:
I think generally it is fine to update them either in the same commit, or wait for the script to bump them later.

Here, I made the changes directly, so that `git grep enablebip61` doesn't return these.

Reply:
yes it's okay to do this (but also to not do this, for ex. because it can cause easily avoidable merge conflicts)